blueocean-dist/pom.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I am inclined to the latter - this may make more sense in the aggregator module. 

A dockerfile in the root would make sense for this IMO. (the only downside is, in some cases, we use the Dockerfile to specify the environment needed for building the project itself, not how to distribute, so it isn't quite as clear cut to me).